@plotly/dash-test-components/src/components/FragmentComponent.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
```

Comment:
Adding a set of components with simple behaviors corresponding to various usage scenarios. This is a top-level component without `n_clicks`